testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/assertions/ElementAssertj.java

```
@@ -246,7 +245,7 @@ private void hasElementInState(
 
     try {
       Awaitility.await()
-          .ignoreException(CamundaClientNotFoundException.class)
+          .ignoreExceptions()
```

❓Do we want to ignore all exceptions? The previous version was ignoring `CamundaClientNotFoundException` because it might be a temporary issue. But after the refactoring we now ignore all types of exceptions.